docs/comparison.md

```diff
@@ -102,6 +102,29 @@ Cons:
 - Blackbox: no explanation for how they manage the patch/PR relationship
 - Dependent on Swift support for your platform
 
+## Jujutsu
+
+[Website](https://github.com/martinvonz/jj)
+
+Pros:
+- When a commit is rewritten, descendants are automatically rebased
+- Supports undo, including undo of a past operation
+- Simpler CLI than `git` (e.g. no "index")
```

Comment:
Side question: Is there a solution for those of us that use the index to commit subsets of our working tree?  Otherwise, this is both a pro and a con :)

Reply:
Yes, the history-editing features mentioned elsewhere in the list are useful for that. Since the working copy is a commit in jj, you can use those commands on the working copy too. For example:

`jj split` will let you interactively split up the working copy in two commits, so that's the primary command for doing "partial commits".

`jj squash/amend -i` will let you interactively move content changes into the parent commit. When you used on the working copy, that's like `git add -i; git amend`.

`jj restore -i` will let you interactively restore paths from another commit. When restoring into the working copy, that's like `git checkout -i` (I think, though I haven't used that in a long time).